domains/omshahane.json

```diff
@@ -1,11 +1,10 @@
 {
-  "description": "Om Shahanee' personal developer website",
-  "repo": "https://github.com/shahane806/shahane806.github.io",
   "owner": {
     "username": "shahane806",
-    "email": "om.p.shahane@gmail.com"
+    "email": "om.p.shahane@gmail.com",
+    "discord": ""
```

Comment:
If you don't want to add discord then you can just remove the field, lemme do it rq